android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireViewModel.kt

```diff
@@ -165,27 +164,7 @@ constructor(
     questionnaire: Questionnaire,
     questionnaireConfig: QuestionnaireConfig
   ) {
-    questionnaireResponse.questionnaire = "${questionnaire.resourceType}/${questionnaire.logicalId}"
-
-    if (questionnaireResponse.logicalId.isEmpty()) {
-      questionnaireResponse.id = UUID.randomUUID().toString()
-      questionnaireResponse.authored = Date()
-    }
-
     viewModelScope.launch(dispatcherProvider.io()) {
-      questionnaire.useContext.filter { it.hasValueCodeableConcept() }.forEach {
-        it.valueCodeableConcept.coding.forEach { coding ->
-          questionnaireResponse.meta.addTag(coding)
-        }
-      }
-
-      // important to set response subject so that structure map can handle subject for all entities
-      handleQuestionnaireResponseSubject(
-        questionnaireConfig.resourceIdentifier,
-        questionnaire,
-        questionnaireResponse
-      )
-
```

Comment:
Any reason why this is deleted? Could it have happened while attempting to resolve merge conflicts?